### PR TITLE
new(ci): move perf CI to a composite action.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

It will be ran by perf CI on PRs and pages CI on master. 
Also, add a new gh pages section with flamegraphs (svgs) built from master.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/generate_inline_svg_md.py
Comment (FedeDP) [inline .github/generate_inline_svg_md.py:1]: Python script to inline an svg file inside a markdown.

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -17,6 +17,51 @@ jobs:
     uses: ./.github/workflows/reusable_kernel_tests.yaml
     secrets: inherit
 
+  perf-libs-master:
```
Comment (FedeDP) [inline .github/workflows/pages.yml:20]: `pages` CI is now responsible to run perf on master.
Moreover, it generates flamegraph svg files using https://github.com/brendangregg/FlameGraph, and uploads it to be later consumed by `deploy-pages` job.

File: .github/workflows/perf.yml
Comment (FedeDP) [inline .github/workflows/perf.yml:1]: Extracted perf related steps to the new composite action.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by jasondellaluce — APPROVED** [review]

**Pull request merged**